CHANGELOG.md

```diff
@@ -2,19 +2,29 @@
 
 ## Unreleased Changes
 
-- Added rate limiting to tool calls: maximum 5 calls per 20 second.
-- Fixed zod version mismatch that caused errors during parameterized tool calls.
-- Refactored environment variable handling to remove `dotenv` dependency from production code in favour of [--env-files](https://nodejs.org/docs/v24.5.0/api/environment_variables.html#env-files).
+## 1.0.0
+
+**Highlights**:
+
+🧠 Davis Analyzers integration for advanced forecasting and anomaly detection
+⚡ Rate limiting and performance improvements
+🔧 Streamlined environment variable handling
```

Comment:
Here's an odd thing I noticed (also in older changelog entries): when using emojis for itemization, they will not have a line break. Should we prefix it with a markdown itemization?

```suggestion
- 🧠 Davis Analyzers integration for advanced forecasting and anomaly detection
- ⚡ Rate limiting and performance improvements
- 🔧 Streamlined environment variable handling
```

Tbh, I could also live without the emojis.

Reply:
Release Notes without Emojis? In what world are we living?